force-app/main/default/classes/OpenTasksCountTriggerHandler.cls

```diff
@@ -0,0 +1,119 @@
+/**
+ * @description       :
+ * @author            : Ferdi AKIN
+ * @group             :
+ * @last modified on  : 02-23-2023
+ * @last modified by  : Ferdi AKIN
+ **/
+public with sharing class OpenTasksCountTriggerHandler {
+  static Set<Id> accIdsSet = new Set<Id>();
+  static ID recordTypeAccount = Schema.SObjectType.Account.getRecordTypeInfosByName()
+    .get('United States')
+    .getRecordTypeId();
+
+  public static void handlerAfterInsert(List<Task> newTasks) {
+    for (Task ts : newTasks) {
+      if (ts.WhatId != null) {
```

Comment:
Assuming I insert tasks for a custom object, then this trigger would consume a query to find out that nothing needs to be done. 

Consider:

```
  private static Set<Id> filterTasks(List<Task> tasks) {
    Set<Id> whatIds = new Set<Id>();
    for (Task task : tasks) {
      if (task.WhatId?.getSobjectType() == Account.SObjectType) {
        whatIds.add(task.WhatId);
      }
    }
    return whatIds;
  }